org.srb2.SRB2.yaml

```
@@ -53,6 +53,33 @@ modules:
           stable-only: true
           url-template: https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-$version+release.autotools.tar.gz
 
+  - name: sdl-mixer-x
+    buildsystem: cmake
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/*.so
+      - /lib/pkgconfig
+      - /share/doc
+    sources:
+      - type: git
+        url: https://github.com/WohlSoft/SDL-Mixer-X
+        commit: 7181c771aa29deaea8bfbe6098613c3a929e9189
```

To track the latest tagged version automatically:

```suggestion
        tag: 2.6.0
        commit: 7181c771aa29deaea8bfbe6098613c3a929e9189
        x-checker-data:
          type: git
          tag-pattern: ^([\d.]+)$
```

---

Done